taskcluster/ci/rust/kind.yml

```diff
@@ -21,13 +21,13 @@ jobs:
     description: 'Rust: Clippy'
     run:
       commands:
-        - ['.', './taskcluster/scripts/rust/rustup-setup.sh'] # this should be a pre-command, but those don't run in the same shell (yet)
+        - ['.', './taskcluster/scripts/rustup-setup.sh'] # this should be a pre-command, but those don't run in the same shell (yet)
```

Comment:
If you're `source`-ing as opposed to running the script itself, it should work, even if it's running in the `pre-command`. That's because https://github.com/mozilla/glean/blob/master/taskcluster/glean_taskgraph/job.py#L62 is basically a chain of commands.

Reply:
Oh, good point. These commands were from the previous integration and we didn't change it yet.